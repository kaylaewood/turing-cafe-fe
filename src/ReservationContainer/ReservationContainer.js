import React, { Component } from 'react';
import './ReservationContainer.css';
import ReservationCard from '../ReservationCard/ReservationCard'

class ReservationContainer extends Component {
  constructor() {
    super()
    this.state = {
      reservations: []
    }
  }

  componentDidMount() {
    fetch('http://localhost:3001/api/v1/reservations')
    .then(response => response.json())
    .then(data => {
      this.setState({
        reservations: data
      })
    })
  }

  <ReservationCard />

  render() {
    return (
      <section className='reservation-container'>
      </section>
    )
  }
}

export default ReservationContainer;
