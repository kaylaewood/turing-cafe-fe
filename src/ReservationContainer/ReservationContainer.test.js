import React from 'react';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('ReservationContainer', () => {
  let reservations = [
    {id: 12, name: 'Trisha', date: '12/09', number: '4'},
    {id: 3, name: 'Heather', date: '12/11', number: '8'}
  ]
  it('should match snapshot as expected', () => {
    const wrapper = shallow(<ReservationContainer
      reservations={reservations}
    />)

    expect(wrapper).toMatchSnapshot();
  })
})
