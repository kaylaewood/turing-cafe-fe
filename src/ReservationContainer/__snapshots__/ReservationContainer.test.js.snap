// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationContainer should match snapshot as expected 1`] = `
<section
  className="reservation-container"
>
  <ReservationCard
    date="12/09"
    id={12}
    key="12"
    name="Trisha"
    number="4"
  />
  <ReservationCard
    date="12/11"
    id={3}
    key="3"
    name="Heather"
    number="8"
  />
</section>
`;
